### PR TITLE
[ZH] Add `Interface/Multiplayer`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] ~Update images (size fix)~

## Changed files

- Client/Interface/Multiplayer/zh.md

### CODE REVIEW

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
+---
+needs_cleanup: true
+---
+
+<!-- TODO: 仍需进一步重写：https://github.com/ppy/osu-wiki/issues/7165 -->
+
+# 多人游戏
+
+**多人游戏** （**Multiplayer**，有时简写为 *Multi*）是一个游戏模式，最多允许 16 位玩家参加，他们可以单独或组队对抗，也可以一起合作游玩，游玩的谱面由房主决定。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:9]:
```suggestion
**多人游戏** （**Multiplayer**，有时简写为 *Multi*）是最多允许 16 位玩家进行个人对抗、团队对抗或合作游玩的游戏模式，游玩的谱面由房主决定。
```

我觉得**概述句**不宜用太多不同的主语，主要信息集中到一句话就好。

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+
+[osu! 学院](/wiki/Community/Video_series/osu!academy)的[第 6 集 (YouTube)](https://www.youtube.com/watch?v=QPTLyG7O8ak) 视频中包括了这部分内容，除此之外还介绍了[在线用户面板](/wiki/Client/Interface/Chat_console#扩展聊天面板)的相关内容。
+
+## 游玩方法
+
+*注意：需要 [osu! 账号](/wiki/Registration)来参加多人游戏，[已禁言](/wiki/Silence)的玩家无法参加。*
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:15]:
```suggestion
*注意：需要 [osu! 账号](/wiki/Registration)来参加多人游戏，[被禁言](/wiki/Silence)的玩家无法参加。*
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+
+按下 `1. Start Game（开始游戏）` 按钮会建立比赛房间，将正在播放的歌曲谱面用作房间谱面。`2. Cancel（取消）` 按钮会将玩家带回到大厅。
+
+## 比赛配置页
+
+::: Infobox
+![](img/multi-room.jpg "玩家视角下的比赛房间")
+:::
+
+在创建或进入比赛房间后，会显示比赛配置页。本文下面几节会按从上到下、从左到右的顺序，着重讲解此页面的不同部分。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:92]:
```suggestion
在创建或进入比赛房间后，会显示比赛配置页。下文会按从上到下、从左到右的顺序，着重讲解此页面的各个部分。
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+
+::: Infobox
+![](img/multi-match-current-players-list.jpg "房间中的玩家列表例图")
+:::
+
+玩家列表会显示房间的所有槽位。`当前玩家` 右侧的括号中会显示房间里的玩家数与已解锁槽位总数。
+
+点击可用槽位即可进行移动，但无法移动到带有上锁图标的槽位。房主可以通过点击槽位左侧的图标将其上锁、解锁，也可以踢走槽位中的玩家。如果要转移房主权限，可以右击目标玩家，然后按下 `将房主交给此玩家` 按钮。在 Team VS 模式中可以将队伍颜色改为蓝色或红色。
+
+光标悬停在表中玩家上时，会显示对应玩家的等级、国家或地区及[整体准确率](/wiki/Gameplay/Accuracy)。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:112]:
```suggestion
光标悬停在表中玩家上时，会显示对应玩家的等级、国家或地区和[整体准确率](/wiki/Gameplay/Accuracy)。
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+
+列表中的玩家栏可以有四种不同颜色，各有不同含义：
+
+| 颜色 | 描述 |
+| :-: | :-- |
+| **红色（没有地图）** | 玩家没有谱面，在下载好谱面前都会显示 `[没有地图]` 通知。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:118]:
```suggestion
| **红色（没有此图）** | 玩家没有谱面，在下载好谱面之前都会显示 `[没有此图]` 。 |
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+| **白色（未准备好）** | 玩家拥有此谱面但还没准备好。在此状态下可以更改[游戏模组](/wiki/Gameplay/Game_modifier)选择。 |
+| **绿色（已准备好）** | 玩家已准备好，此时不能更改模组选择。在有玩家准备好后，房主就可以开始比赛，所有已准备好的玩家都会开始游玩。 |
+| **浅蓝色（游玩中）** | 玩家正在游玩，在比赛结束前都会显示 `[游玩中]` 通知。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:121]:
```suggestion
| **浅蓝色（游玩中）** | 玩家正在游玩，在比赛结束前都会显示 `[游玩中]` 。 |
```

`notice`不太好翻译啊，干脆直接扔了😎

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 7]
```diff
@@ -0,0 +1,405 @@
...
+| `OD` | 判定严度 |
+| `HP` | 掉血速度 |
+| `Stars` | 星数评价 |
+
+如果玩家没有对应谱面，则会给出如下状态之一：
+
+| 谱面状态 | 描述 |
+| :-: | :-- |
+| `Ranked` / `Approved` / `Pending` / `Graveyard` | [谱面类别](/wiki/Beatmap/Category)。点击谱面卡片后会打开谱面信息页，可以在这里[下载](/wiki/Beatmap#下载谱面)它。 |
+| `Not uploaded or not up-to-date（未上传或非最新）` | 无法下载此谱面。如果问题仍然存在，房主应该帮玩家找到（[谱面列表](https://osu.ppy.sh/beatmapsets)中的）链接。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:167]:
```suggestion
| `Not uploaded or not up-to-date（未上传或非最新）` | 无法下载此谱面。如果问题无法解决，房主应该帮玩家找到（[谱面列表](https://osu.ppy.sh/beatmapsets)中的）链接。 |
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 8]
```diff
@@ -0,0 +1,405 @@
...
+
+*对于关于组队模式的更多信息，参见[组队模式游玩小节](#team-mode-gameplay).*
+
+可以使用以下四种组队模式进行比赛：
+
+| 组队模式 | 描述 |
+| :-- | :-- |
+| `Head-to-head` | 相互比拼，竞争榜首。 |
+| `Team VS` | 组队（红队和蓝队）比拼，获得胜利。 |
+| `Tag co-op` （仅限 osu! 模式，不计入排名） | 一起完成谱面，一次打一串连击。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:190]:
```suggestion
| `Tag co-op` （仅限 osu! 模式，不计入排名） | 一起完成谱面，每位玩家一次打一组连击。 |
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 9]
```diff
@@ -0,0 +1,405 @@
...
+| `Tag-team VS` （仅限 osu! 模式，不计入排名） | 与 `Tag co-op` 相同，但是有两支队伍相互对立。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:191]:
```suggestion
| `Tag-team VS` （仅限 osu! 模式，不计入排名） | 与 `Tag co-op` 相同，但是有两支队伍相互对抗。 |
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 10]
```diff
@@ -0,0 +1,405 @@
...
+
+##### Tag 颜色
+
+::: Infobox
+![](img/multi-tag-colours.jpg "玩家负责连击组的可用颜色")
+:::
+
+如果组队模式是 `Tag co-op` 或 `Tag-team VS`，则会显示 `Tag Colour（Tag 颜色）: (combo colour)` 小节，让每位玩家可以选择自定义颜色。如果颜色设置为 `Default（默认）`，则会使用谱面原有的连击组颜色。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:199]:
```suggestion
如果组队模式是 `Tag co-op` 或 `Tag-team VS`，则会显示 `Tag Colour（Tag 颜色）: (combo colour)` 选项，让每位玩家可以选择自定义颜色。如果颜色设置为 `Default（默认）`，则会使用谱面原有的连击颜色。
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 11]
```diff
@@ -0,0 +1,405 @@
...
+| `Score（分数）` | 分数最高的玩家获胜。 |
+| `Accuracy（准确率）` | 准确率最高的玩家获胜。如果准确率为 100.00% 的玩家不止一位，则（转盘获得）分数最高的玩家获胜，否则算作平局。 |
+| `Combo（连击数）` | *在谱面结束时*连击数最高的玩家获胜。如果连击数相同，则分数最高的玩家获胜。此条件下不考虑最高连击数。 |
+| `Score v2` | （标准化）分数最高的玩家获胜。 |
+
+### 比赛设置按钮
+
+[聊天面板](/wiki/Client/Interface/Chat_console)上方有两个大按钮，分别是橙色和蓝色的。
+
+左边的橙色 `离开房间` 按钮的作用不言自明。当房主离开房间时，房主资格会按槽位自动转移给剩余玩家。房间里没有玩家时会自动关闭，但由 [`!mp` 命令](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)建立的房间除外，这类房间会在 30 分钟后关闭。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:216]:
```suggestion
左边的橙色 `离开房间` 按钮的作用顾名思义。当房主离开房间时，房主资格会按槽位自动转移给剩余玩家。房间里没有玩家时会自动关闭，但由 [`!mp` 命令](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)建立的房间除外，这类房间会在 30 分钟后关闭。
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 12]
```diff
@@ -0,0 +1,405 @@
...
+
+蓝色按钮用于控制玩家的准备状态，也可以使比赛开始。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:218]:
```suggestion
蓝色按钮用于控制玩家的准备状态，以及开始比赛。
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 13]
```diff
@@ -0,0 +1,405 @@
...
+
+无法暂停多人游戏。按下 `Esc` 键后会在右下角给出警告，再按一次则会退出比赛房间。
+
+#### 视觉设置
+
+在比赛将开始时，将光标移动到屏幕底端会打开视觉设置面板。对于更多信息，参见[视觉设置](/wiki/Client/Interface/Visual_settings)页面。
+
+#### 血量
+
+当玩家的血条完全清空时，玩家仍然可以继续游玩，但会被视作失败，其分数不会出现在排行榜上。在没有使用 [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death) 游戏模组的情况下，玩家在血条回满时就会复活。
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:251]:
```suggestion
当玩家的血条掉光时，玩家仍然可以继续游玩，但会被视作失败，其分数不会出现在排行榜上。在没有使用 [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death) 游戏模组的情况下，玩家在血条回满时就会复活。
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 14]
```diff
@@ -0,0 +1,405 @@
...
+
+在 Team VS 模式中，比赛结束时处于失败状态的玩家，其分数不会计入所在队伍总分。如果队伍中的所有玩家都失败了，比赛就会立刻结束，默认另一队获胜。
+
+#### 结果
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:255]:
```suggestion
#### 结算
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 15]
```diff
@@ -0,0 +1,405 @@
...
+
+如果玩家在比赛中途退出，游戏则会完美游玩他们负责的部分，但不会计入所属队伍。如果一支队伍中的所有玩家都退出了，默认另一支队伍获胜。
+
+失败或已退出的玩家位于排行榜底部，其分数与其他玩家分开排列。
+
+游玩时，排行榜中玩家的背景颜色指示其状态。
+
+| 图例 | 状态 | 描述 |
+| :-: | :-: | :-- |
+| ![](img/Playerbox/Multi_PB_B.jpg "正常") | 正常 | 玩家剩余血量超过一半。取决于血量变化，颜色可能会变亮，也可能变成红色。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:293]:
```suggestion
| ![](img/Playerbox/Multi_PB_B.jpg "正常") | 正常 | 玩家剩余血量超过一半。颜色会根据血量变为亮蓝色或亮红色。 |
```

File: wiki/Client/Interface/Multiplayer/zh.md [hunk 16]
```diff
@@ -0,0 +1,405 @@
...
+| ![](img/Playerbox/Multi_PB_R.jpg "危险") | 危险 | 玩家剩余血量不到一半。取决于血量变化，颜色可能会变亮，也可能变成蓝色。 |
```
Comment (Proxyerium) [inline wiki/Client/Interface/Multiplayer/zh.md:294]:
```suggestion
| ![](img/Playerbox/Multi_PB_R.jpg "危险") | 危险 | 玩家剩余血量少于一半。颜色会根据血量变为亮蓝色或亮红色。 |
```

**Review by Proxyerium — COMMENTED** [review]
这回看完了

**Review by Proxyerium — APPROVED** [review]

**Review by TicClick — CHANGES_REQUESTED** [review]
please retake the screenshots, they need to be 1280x720: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#screenshots-of-gameplay

**Comment by CloneWith** [comment]
@TicClick
have planned for image updating, but busy with other work 🔥 
so I deleted those images and may upload them later in another PR

**Review by TicClick — APPROVED** [review]

**Pull request merged**